### DISCUSSION TITLE
When will the "preview" of s3-transfer-manager end?

### DISCUSSION THREAD

**Discussion by raymondchen625** [aws/aws-sdk-java-v2#3504] [General]
Hi, I noticed `s3-transfer-manager` has been i[n preview](https://github.com/aws/aws-sdk-java-v2/blob/a634a015265412d0488b720f816a5989713c77cc/services-custom/s3-transfer-manager/pom.xml#L28) for over 15 months. Wondering if there is a plan to make it official. Is there any blocking issues users should be aware of?

**Comment by debora-ito** [comment]
Although we don’t comment on release schedules in general, we can say that we are actively working on the release path of TransferManager v2 as a priority.

This week we conducted performance tests, and we have a short list of remaining task to complete ahead of GA.

We appreciate your patience, and any feedback you have on the TransferManager v2 so far.
